packages/loader/container-loader/src/prefetchDocumentStorageService.ts

```diff
@@ -27,7 +27,11 @@ export class PrefetchDocumentStorageService extends DocumentStorageServiceProxy
         return p;
     }
 
-    public async read(blobId: string): Promise<string> {
+    /**
+     *
+     * @deprecated - only here for back compat, will be removed after release
+     */
+    public async read(blobId: string): Promise<ArrayBufferLike > {
```

Comment:
Please be consistent - everything that returns blobs should be called readBlob, everything that returns base64-encoded string should be called read(). Mixing names / return types will confuse developers reading code

Reply:
Besides that, compiler should fail here as it should not match signature on interface. If it does not, worth drilling on why, as that might be hiding other bugs in other places - compiler should help us in making sure code is correct

Reply:
I see! So what should all deprecated read return? does it matter? should they return something like `bufferToString(readBlob(blobId))`?